content/learn/text-editing.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Text-Editing Workflow
+weight: 6800
+type: page
+abstract:  "Our process for adding publication content and data into Quire via text editor"
+online: false
+---
+
+{{< q-checklist >}}
+
+- [ ] Open project in text editor
+- [ ] Delete starter content
+- [ ] Complete the `publication.yml`
+- [ ] Use publication TOC to structure placeholder Markdown files and landing pages
+- [ ] Add publication content as Markdown (`.md`) files in the `content` folder
```

Comment:
In the step above, you're suggesting they're already creating the files, right? So here, to add publication content, they'd copy and pasted it in to the placeholder pages.